src/components/sbb-link/sbb-link.scss

```diff
@@ -95,6 +98,11 @@ button.sbb-link:disabled {
   }
 }
 
+.sbb-link--icon-placement-start,
+.sbb-link--icon-placement-end {
+  gap: var(--sbb-spacing-fixed-1x);
+}
+
```

Comment:
This change looks unlogical to me, as either of this class is always set. So why can't the gap be part of .sbb-link?

Reply:
It only needs the gap if the content of the link contains more than a text. In this case the icon, in front or at the end.

Reply:
Look at sbb-link.ts:

```
    const iconPositionClass =
      this.iconPlacement === 'start'
        ? ' sbb-link--icon-placement-start'
        : ' sbb-link--icon-placement-end';
```

There is always either of one class set, so your change does not introduce any visible change instead of making more complex code ;-)

Reply:
Rechecked it and could not reproduce the reason why I switched to this approach. I will revert this. :)